src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,32 @@
+.nav-link {
```

Comment:
can we make this camelCase ?

Reply:
i am adding the .nav-link of bootstrap class to the links of the route so it can handle all the breakpoints layout by default 
i add another class to it "navLink" and overwrite the basic one with the new one